### DISCUSSION TITLE
Running sessions with balnk spaces

### DISCUSSION THREAD

**Discussion by ghost** [devlikeapro/waha#141] [Ideas]
I was requesting if it could be possible to allow session names containing spaces to run successfully.
Example:
{
    "name": "Joshua  ",
  }
this session will not run because it has a space at the end. The session status will not change to SCAN_QR_CODE.

BUT this session will run successfully because it has no space
{
    "name": "Joshua",
  }

Please advise if it is possible to implement that.
Thanks

**Comment by allburov** [comment]
I think it's possible, but it's high priority task tho.
Could you create an issue for that so we remember about it please

**Comment by ghost** [comment]
Okay, Thanks

**Comment by allburov** [comment]
The answer about invalid symbols - https://github.com/devlikeapro/whatsapp-http-api/issues/142#issuecomment-1636997983
